kubemarine/cri/containerd.py

```diff
@@ -39,9 +41,42 @@ def enrich_inventory(inventory: dict, _: KubernetesCluster) -> dict:
         containerd_config[runc_options_path]['SystemdCgroup'] = \
             bool(strtobool(containerd_config[runc_options_path]['SystemdCgroup']))
 
+    # Check if field for new and old configuration formats are presented
+    old_format_result, old_format_field = contains_old_format_properties(inventory)
+    new_format_result, new_format_field = contains_new_format_properties(inventory)
+
+    # Fail if fields for both formats are presented
+    if old_format_result and new_format_result:
+        raise errors.FailException(f"Invalid containerd configuration: "
+                                   f"{old_format_field} can't be set when {new_format_field} is provided")
```

Comment:
Would be useful to specify in documentation that two formats could not be used simultaneously

Reply:
added notes in documentation